packages/Webkul/Shipping/src/Config/system.php

```diff
@@ -86,6 +93,13 @@
                 'validation'    => 'required',
                 'channel_based' => false,
                 'locale_based'  => true,
+            ], [
+                'name'          => 'is_calculate_tax',
+                'title'         => 'Calculate Tax',
```

Comment:
Please change this to: Calculate tax on checkout

Reply:
Added the missing translation.